curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/63db7f4677d06d7500a13321.md

```diff
@@ -9,6 +9,8 @@ dashedName: step-1
 
 In this project, you'll build a decimal and binary converter and learn about both number systems. You'll also learn about recursion by using it to perform the conversions.
 
+Note this project will only convert positive numbers into binary.
```

Comment:
i would suggest to add a little bit of logic as to why we only check for positive decimal numbers, and not negative numbers. What do you say @ilenia-magoni  @jdwilkin4

Reply:
So the reason why we are limiting this to positive numbers is because the real goal of this project should be focused on learning recursion. Not an in depth lesson on binary. The binary converter is just a vehicle to teach recursion. So instead of refactoring this app to add include handling negative numbers, it is best to limit it to positive and focus all of the learning on recursion.

So as for this note here, we could remove this sentence and just update the first sentence to say this instead.

> Recursion is a technique used to reduce a complex problem into simpler sub-problems, until they become straightforward to solve. In this project, you will learn about recursion by building an app that takes positive decimal numbers and converter then to the binary number system.